lib/widgets/body/special_experience.dart

```diff
@@ -30,8 +32,8 @@ class _SpecialExperienceState extends State<SpecialExperience>
           my.AnimatedWidget(
             scrollNotifier: widget.scrollNotifier,
             position: widget.logoTop,
-            // TODO: revert to 3447.0
-            startingPoint: 3170.0,
+            // TODO: revert later
+            startingPoint: 3447 - tempOffsetForScrollTrigger,
```

Comment:
I want to discuss and unify how to express double type variable! 😁

Reply:
Good point. I recommend to use constant value itself as double like `3447.0`.
Because it prevents a needless type conversion.

Reply:
👍